modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -160,6 +160,23 @@ object Model {
     case object Gems extends Resource
     case object Altair extends Resource
 
+    implicit val order: Order[Resource] = Order.orderBy {
```

Comment:
Why do we need this order?

Reply:
This is to show the labels on a given order. Otherwise you can get them change position as the executions progress

Reply:
Does this mean that whenever we add a new instrument we need to update this? Why this information is in the "Model"?

Reply:
Certainly we need to update the model in each instrument. This could be moved to the UI, it was just simpler to it there

Reply:
Actually this needs to be in `Model` as we are sending the `Resources` sorted from the server